Userland/Libraries/LibJS/Bytecode/StringTable.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2021, Gunnar Beutner <gbeutner@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJS/Bytecode/StringTable.h>
+
+namespace JS::Bytecode {
+
+StringTableIndex StringTable::insert(const StringView& string)
```

Comment:
`StringView const&`
Although, `StringView` is just two values so it's probably even nicer to pass it by value.